bolt04/onion-test.json

```diff
@@ -0,0 +1,37 @@
+{
+  "comment": "A testcase for a variable length hop_payload. The third payload is 256 bytes long.",
```

Comment:
Looks like I forgot to update this comment, the third payload isn't 256 bytes long!

Reply:
Yep, fixed to "275 bytes long".

Reply:
Apologies for leaving this comment only after the PR has been merged, but I believe the "comment" is still incorrect as it's the fifth payload that is 275 bytes long, no?
There's also the second payload that's 83 bytes long, so there's now two payloads that have a different size than the rest of the payloads, so perhaps it'd be worth mentioning both if we want to explicitly mention the payloads that have a different size. If you'd like me to, I can of course open a PR editing the "comment" :).